Tests/Fluent.Tests/Batch/BatchTests.cs

```diff
@@ -242,5 +245,135 @@ public async Task CanCreateBatchAccountWithApplication()
                 }
             }
         }
+
+        [Fact]
+        public async Task CanCreateBatchAccountWithPool()
+        {
+            using (var context = FluentMockContext.Start(this.GetType().FullName))
+            {
+                try
+                {
+                    var poolId = "testPool";
```

Comment:
Please use random name for testing

Reply:
Should we apply the random name rule to existing test cases in BatchTests.cs? It seems all names there are fixed value rather than generating randomly. We don't have random generator in .net while we do have that in Java for only resource group name, batch name, storage account name.

Reply:
For now CanCRUDBatchAccounts() generated the name randomly while CanCreateBatchAccountWithApplication() didn't. I will change to random name for it and the new added test case.

Reply:
We have random string generator for other test cases. Please check other test cases.

Reply:
done